src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/CodeDirectiveFormattingTest.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting;
 public class CodeDirectiveFormattingTest(FormattingTestContext context, HtmlFormattingFixture fixture, ITestOutputHelper testOutput)
     : FormattingTestBase(context, fixture.Service, testOutput), IClassFixture<FormattingTestContext>
 {
-    [FormattingTestFact(SkipFlipLineEnding = true)]
```

Comment:
>SkipFlipLineEnding [](http://example.com/codeflow?start=24&length=18)

Can this property be removed now?

Reply:
Yeah, I thought about it, but figured leaving it just saves someone adding it back later if we have some unsupported feature or bug. It's not currently used though, so I'm happy to if you think it would be better?

Reply:
I guess I'd just prefer we get rid of it if it's not in use anymore. Seems easy enough to add back if someone needs to later.